eos-dev-recommends

```diff
@@ -1,2 +1 @@
-eos-dev-config
```

Comment:
I actually think we can make it useful by stripping out all the nasty parts, but I don't think it's worth the effort right now.

I do have a pipe dream to restore the `eosdev` ostree that uses the `eos-dev` metapackage and make `eosdev` master images for developers that have useful development utilities on them. At least for me a standard `eos` image is not capable enough for my work.

Reply:
Yep, I'm keen to not get rid of the `eos-dev` metapackage. It would also be a good base for a container image that could be run with `podman`.